sw/device/silicon_creator/lib/drivers/flash_ctrl.c

```diff
@@ -343,12 +347,10 @@ rom_error_t flash_ctrl_info_read(flash_ctrl_info_page_t info_page,
                                  uint32_t offset, uint32_t word_count,
                                  uint32_t *data) {
   const uint32_t addr = info_page_addr(info_page) + offset;
```

Comment:
For cases in which `flash_ctrl_info_write` is used, I think we check the result using `flash_ctrl_info_read`.  It would be good to consider if we need to add redundant `flash_ctrl_info_read` calls for additional protection. However, I believe we already have enhanced coverage in the digest calculation.

Reply:
I agree, we can also revisit this when going through flash_ctrl_info_write usages.